src/modules/auth/clients/ClientCreateContainer.js

```diff
@@ -195,177 +195,173 @@ const ClientCreateContainer = (props) => {
-							/>
-							<URiInput
-								name="redirect_uris_main"
-								templateName="redirect_uris"
-								invalid={errors?.redirect_uris_main && true}
+								valueList={metaData["properties"]["redirect_uri_validation_method"]["enum"]}
 								disabled={disabled}
-								errors={errors}
-								append={append}
-								remove={remove}
-								fields={fields}
+								labelName={t('ClientCreateContainer|Redirect URI validation method')}
+								editing={(client != undefined)}
+							/>}
+						{(client == undefined) &&
+							<TextInput
+								name="preferred_client_id"
 								register={register}
-								reg={regRedirectUrisMain}
-								labelName={`${t("ClientCreateContainer|Redirect URIs")}*`}
+								errors={errors}
+								disabled={disabled}
+								labelName={t('ClientCreateContainer|Preferred client ID')}
 							/>
-							{(metaData != undefined) && metaData["properties"] && metaData["properties"]["redirect_uri_validation_method"] &&
-								<RadioInput
-									key="redirect_uri_validation_method"
-									name="redirect_uri_validation_method"
-									register={register}
-									valueList={metaData["properties"]["redirect_uri_validation_method"]["enum"]}
-									disabled={disabled}
-									labelName={t('ClientCreateContainer|Redirect URI validation method')}
-									editing={(client != undefined)}
-								/>}
-							{(client == undefined) &&
-								<TextInput
-									name="preferred_client_id"
-									register={register}
-									errors={errors}
-									disabled={disabled}
-									labelName={t('ClientCreateContainer|Preferred client ID')}
-								/>
-							}
-							<TextInput name="client_uri" register={register} disabled={disabled} labelName={t('ClientCreateContainer|Client URI')}/>
-						</CardBody>
-						<CardFooter>
-							<ButtonGroup>
-							{((client != undefined) && (editClient == true)) ?
-								<>
-									<ButtonWithAuthz
-										title={t("ClientListContainer|Save")}
-										color="primary"
-										type="submit"
-										disabled={isSubmitting}
-										resource={resource}
-										resources={resources}
-									>
-										{t("ClientListContainer|Save")}
-									</ButtonWithAuthz>
-									<ButtonWithAuthz
-										outline
-										title={t("Cancel")}
-										color="primary"
-										type="submit"
-										disabled={isSubmitting}
-										resource={resource}
-										resources={resources}
-										onClick={() => props.history.push(`/auth/clients/${client_id}`)}
-									>
-										{t("Cancel")}
-									</ButtonWithAuthz>
-								</>
-							:
+						}
+						<TextInput name="client_uri" register={register} disabled={disabled} labelName={t('ClientCreateContainer|Client URI')}/>
+					</CardBody>
+					<CardFooter>
+						<ButtonGroup>
+						{((client != undefined) && (editClient == true)) ?
+							<>
 								<ButtonWithAuthz
-									title={t("ClientCreateContainer|Create")}
+									title={t("ClientListContainer|Save")}
```

Comment:
@petrKavulok Well, this is `ClientCreateContainer`, not `ClientListContainer`

Reply:
I didn't realize 'save' was in the common locales used across all apps, so we're using just `t("Save")` now